bokeh/plotting/_docstring.py

```diff
@@ -117,7 +117,7 @@ def _docstring_other():
         Any explicitly set values for ``line_alpha``, etc. will override this
         setting.
 
-    color (Color, optional) :
+    color (color, optional) :
```

Comment:
This change is necessary because Sphinx now tries to auto-reference `Color`, and it is ambiguous. So we may be able to simplify many docstrings while still providing ref links.